config/200-config.yaml

```diff
@@ -109,3 +109,7 @@ data:
     # Defaults to false. More info Envoy Docs.
     # see: https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/ext_authz/v3/ext_authz.proto.html#extensions-filters-http-ext-authz-v3-buffersettings
     extauthz-pack-as-byte: "false"
+
+    # Specifies the secret that contains the TLS certificate and key pair when using HTTPS communication with Kourier Ingress.
+    certs-secret-name: ""
+    certs-secret-namespace: ""
```

Comment:
I'd like to mention that env vars are still supported, but `ConfigMap` will take priority and override env var values.

Vice versa I wonder if we should add comment to deployment wrt/ possibility to use CM now. More like brainstorming open ended questions. Wdyt?

Reply:
I agree. Without reading the code, the current behaviour isn't clear, so I’ve added comments to note that ConfigMap values override environment variables.

In theory, mentioning this in the deployment makes sense, but in practice, comments there get lost during 'kubectl apply', and I suspect few users notice them.

Also, is using environment variables considered deprecated in Kourier?
If so, we could make that clearer, perhaps with a log message when environment variables are used, or by noting it in the README.